src/components/BoardSizeForm.tsx

```diff
@@ -1,29 +1,33 @@
 import * as React from 'react';
+import { Input } from './atoms/StyledInput';
 import Error from './Error';
+import { ThemeProvider } from 'styled-components';
+import theme from './Theme';
 
 const BoardSizeForm: React.SFC<BoardFormProps> = props => {
   const { onChange, onSubmit, hasError } = props;
+
   return (
     <React.Fragment>
       <p>What size should the board be?</p>
       <form className='size-form' onSubmit={onSubmit}>
-        Board Size:
-        <input
+        <Input
+          placeholder='board size'
           onChange={onChange}
           type='text'
           name='size'
           aria-label='board size input'
         />
-        <input className='size-submit' type='submit' value='submit' />
+        <Input submit className='size-submit' type='submit' value='submit' />
```

Comment:
Is `submit` a valid property? I'm not aware of it.

Reply:
Maybe I'm confused about how to hand a styled component a prop to change how it renders.

I was trying to hand it a prop like 
`<Button primary>Primary Button</Button>`

that would change how the styled component renders:
```
...
${props => props.primary && css`
    background: palevioletred;
    color: white;
  `}
...
```

Reply:
Ohh, nope, that's totally correct! 👍 My bad.